src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -472,6 +473,23 @@ internal string GetIntputOutputTrace(string rawInputOutputData)
             return this.httpApiHandler.CreateCheckStatusResponse(request, instanceId, attribute);
         }
 
+        // Get a response that will wait for response from the durable function for predefined period of time before 
+        // pointing to our webhook handler. 
+        internal async Task<HttpResponseMessage> CreateCheckStatusResponse(
+            HttpRequestMessage request,
+            string instanceId,
+            OrchestrationClientAttribute attribute,
+            TimeSpan timeout,
+            TimeSpan retryInterval)
+        {
+            if (DisableHttpManagementApis)

```

Comment:
Please prefix with `this.` to make it clear to the reader that this is an instance member.

Reply:
Sure.